aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -433,9 +438,9 @@ Tensor& arctanh_out(const Tensor& self, Tensor& result) { return at::atanh_out(r
 Tensor arctanh(const Tensor& self) { return self.atanh(); }
 Tensor& arctanh_(Tensor& self) { return self.atanh_(); }
 
-Tensor& square_out(const Tensor& self, Tensor& result) { return at::pow_out(result, self, 2); }
-Tensor square(const Tensor& self) { return at::pow(self, 2); }
-Tensor& square_(Tensor& self) { return self.pow_(2); }
+// FIXME: remove const_cast once pow_out is ported to structured
```

Comment:
Actually, `pow` is already structured. The const_cast is needed because the C++ API signatures generated in `build/aten/src/ATen/Functions.h` all return mutable `Tensor&` instead of `const Tensor&`. We have an open issue for it here: https://github.com/pytorch/pytorch/issues/54001